src/webapp/components/dataset-wizard/SummaryDataSet.tsx

```diff
@@ -75,6 +76,16 @@ export const SummaryList = React.memo((props: { dataSet: DataSet; orgUnits: OrgU
         .map(indicator => indicator.coreCompetency?.name || "")
         .join(", ");
 
+    const regionsCodes = _(dataSet.access)
+        .filter(access => access.type === "groups")
+        .compactMap(access => {
+            return access.name.split("_")[0];
```

Comment:
This kind of processing the view as a bit of a code smell, do we need to know this here?